README.md

```diff
@@ -1,3 +1,6 @@
 # parcel-examples
 
 🛠 Parcel examples using various frameworks
+
+## [Vue.js](https://vuejs.org)
+- [Parcel Basic Example with Vue.js](https://github.com/parcel-bundler/examples/Vue)
```

Comment:
Just `Vue/' would work fine: `[Parcel Basic Example with Vue.js]('Vue/')`

Reply:
Happy? 

- [x] Done in ee2390e

Reply:
- [x] relative `Vue/` in 4cfe523